dashboard/config/scripts/levels/CSD U3 AnimationsMulti introducing multiframe.level

```diff
@@ -133,10 +133,10 @@
       "console.log": null,
       "functionParams_none": null,
       "callMyFunction": null,
-      "comment": null
+      "comment": null,
+      "stopSound": null
     },
     "show_debug_watch": "false",
-    "encrypted_examples": "hBYEo3wudGBFMFzTZ/joDBvBc0h7k5p7JWdsCu9nmhxMPKlb7gdxA2UfwmPx\nbdwa\n",
```

Comment:
no idea. Maybe I need to have some special keys in my environment? I don't know how these work.